tests/test_app.py

```diff
@@ -22,10 +22,25 @@
 from firebase_admin import _utils
 from tests import testutils
 
-
 CREDENTIAL = credentials.Certificate(
     testutils.resource_filename('service_account.json'))
 GCLOUD_PROJECT = 'GCLOUD_PROJECT'
+CONFIG_JSON = firebase_admin._CONFIG_JSON_ENV
+
+# This fixture will ignore the environment variable pointing to the default
+# configuration for the duration of the tests.
+@pytest.fixture(scope="session", autouse=True)
```

Comment:
What are the tests affected by this fixture? Ideally it shouldn't affect tests in other modules.

Reply:
yes you are right, this is no longer needed.